lib/MusicBrainz/Server/Controller/Role/Collection.pm

```diff
@@ -109,11 +109,16 @@ View a list of collections that this work has been added to.
 
         $c->model('Editor')->load(@public_collections);
 
-        $c->stash
-          (entity_type => $entity_type,
-           public_collections => \@public_collections,
-           private_collections => $private_collections,
-           template => 'entity/collections.tt',
+        my %props = (
+            entity => $c->stash->{$entity_name // $entity_type}->TO_JSON,
```

Comment:
`$entity_name` is already defined as `my $entity_name = $params->entity_name // $entity_type;` so it doesn't seem like the fallback is needed here?
```suggestion
            entity => $c->stash->{$entity_name}->TO_JSON,
```